lib/qiita/markdown/filters/external_link.rb

```diff
@@ -0,0 +1,35 @@
+module Qiita
+  module Markdown
+    module Filters
+      class ExternalLink < HTML::Pipeline::Filter
+        def call
+          return doc unless hostname
```

Comment:
This should be validated in [`HTML::Pipeline::Filter#validate`](https://github.com/jch/html-pipeline/blob/f6124861a97a5a0dd0f496d720c1b1392e4ffe37/lib/html/pipeline/filter.rb#L78-L80) like [this](https://github.com/jch/html-pipeline/blob/f6124861a97a5a0dd0f496d720c1b1392e4ffe37/lib/html/pipeline/emoji_filter.rb#L33-L37).